portal-web/test/functional/com/liferay/portalweb/macros/Form.macro

```diff
@@ -916,9 +916,9 @@ definition {
 	}
 
 	macro viewFieldsGroupCollapsedField {
-		AssertElementNotPresent(
+		AssertElementPresent(
 			key_fieldLabel = "${fieldLabel}",
-			locator1 = "Form#FIELDS_GROUP_FIELD_COLLAPSIBLE_IS_VISIBLE");
+			locator1 = "Form#FIELDS_GROUP_COLLAPSED_FIELD");
```

Comment:
@leticiachiu, shouldn't we perform the same change in macro `viewFieldsGroupNotCollapsedField`? If so, will we still need the path `FIELDS_GROUP_FIELD_COLLAPSIBLE_IS_VISIBLE`?

Reply:
I didn't change the `viewFieldsGroupNotCollapsedField` because it's working with `FIELDS_GROUP_FIELD_COLLAPSIBLE_IS_VISIBLE`, but I can use this new path for both macros too :)) thanks for the suggestion!